entrypoint.sh

```diff
@@ -71,6 +71,14 @@ parse_args() {
   return 0
 }
 
+override_python_packages() {
+  echo "Overrides: ${OVERRIDE}"
+  for package in ${OVERRIDE}
```

Comment:
It should be possible to do
```suggestion
  pip install ${OVERRIDE}
```
and this is even better because it'll show cross-dep conflicts.

Also, we should probably add
```sh
pip check
```
afterward

Reply:
How would you like to see the pip check done?